test/common/http/filter/lua/lua_filter_test.cc

```diff
@@ -1171,7 +1171,7 @@ TEST_F(LuaHttpFilterTest, ImmediateResponse) {
   const std::string SCRIPT{R"EOF(
     function envoy_on_request(request_handle)
       request_handle:respond(
-        {[":status"] = "403"},
+        {[":status"] = "503"},
```

Comment:
Why change this test response code? Both are in range..

Reply:
Because this was what was broken before the change. :)

We test 4xx codes elsewhere.